src/main.ts

```diff
@@ -36,4 +36,8 @@ settingsInit()
 .then(themeInit)
 .then(i18nInit)
 .then(() => processWindowLocation())
-.then(() => Plugins.SplashScreen.hide())
+.then(() => {
+  setTimeout(() => {
```

Comment:
why adding this timeout?

Reply:
I [searched](https://www.google.com/search?q=capacitor+white+screen+after+splash) for this problem and several people recommended setting a small timeout to avoid the flash. This should mitigate a black flash in light mode - I'll upload a screen capture

Reply:
yes I guess it should be fine like that; and it's better to have a black background than a white one.

Reply:
https://user-images.githubusercontent.com/569991/125131051-1d0ad280-e0d0-11eb-9e3b-f91b7e53f953.mov